patches/@codspeed+core+3.1.0.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/node_modules/@codspeed/core/dist/index.cjs.js b/node_modules/@codspeed/core/dist/index.cjs.js
+index 1c40cda..4a5d588 100644
+--- a/node_modules/@codspeed/core/dist/index.cjs.js
++++ b/node_modules/@codspeed/core/dist/index.cjs.js
+@@ -26,7 +26,10 @@ const getV8Flags = () => {
+     "--no-opt",
+     "--predictable",
+     "--predictable-gc-schedule",
+-    "--interpreted-frames-native-stack"
++    "--interpreted-frames-native-stack",
++    // "--jitless",
++    '--no-concurrent-sweeping',
++    '--max-old-space-size=4096',
+   ];
+   if (nodeVersionMajor < 18) {
+     flags.push("--no-randomize-hashes");
```

Comment:
As recommended by codspeed maintainers

Reply:
For existing benchmark suite, we have the following node options:

      '--predictable',
      '--no-concurrent-sweeping',
      '--no-minor-gc-task',
      '--min-semi-space-size=1024', // 1GB
      '--max-semi-space-size=1024', // 1GB
      '--trace-gc', // no gc calls should happen during benchmark, so trace them

Do we want to use similar flags? Are they equivalent? Asking from a place of ignorance here.

Tiny nit in terms of uniform quoting for the options, double quote vs single quote, just because I can't help myself. :)

Reply:
And regarding the quotes, since this is a patch applied to codspeed-core, I tried to align with the code style in that folder.